src/wallet/rpcwallet.cpp

```diff
@@ -1186,6 +1186,9 @@ UniValue sendmany(const JSONRPCRequest& request)
     if (totalAmount > nBalance)
         throw JSONRPCError(RPC_WALLET_INSUFFICIENT_FUNDS, "Account has insufficient funds");
 
+    // Shuffle recipient list
+    std::shuffle(vecSend.begin(), vecSend.end(), FastRandomContext());
```

Comment:
12709 should be skipped completely, it's just a waste of cpu and has no effect for us because `CreateTransaction` will sort ins/outs according to BIP69 anyway.

Reply:
dropped 12709 and 12193